sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/antlr4/imports/mysql/MySQLDDLStatement.g4

```diff
@@ -40,7 +40,7 @@ dataType
     ;
 
 dataTypeName_
-    : ID ID?
+    : uid uid?
```

Comment:
why data  type composite with tow uid? maybe should make code more clear here

Reply:
uid match more than ID